src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/util/InterventionFactory.kt

```diff
@@ -1,8 +1,7 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.util
 
 import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
-import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.DynamicFrameworkContract
-import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Intervention
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.*
```

Comment:
I think wildcards this will fail ktlint checks

Reply:
ah, damn intellij, my settings are wrong. thanks.